wallet/src/wallet/mod.rs

```diff
@@ -970,8 +969,8 @@ impl<B: storage::Backend> Wallet<B> {
             let tx = signer
                 .sign_tx(ptx, account.key_chain())
                 .map(|(ptx, _, _)| ptx)?
-                .into_signed_tx(chain_config)
-                .map_err(error_mapper)?;
+                .into_signed_tx()
+                .map_err(|e| error_mapper(WalletError::TransactionCreation(e)))?;
```

Comment:
we should verify the signatures after this, just in case the user selected a multisig utxo as an input, in that case the input will not be fully signed.

Reply:
add verification code